packages/styling/src/styles/theme.ts

```diff
@@ -217,14 +217,19 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depCom
     inputBorderHovered: p.neutralPrimary,
     inputBackground: p.white,
     inputBackgroundChecked: p.themePrimary,
+    inputBackgroundHovered: p.neutralLighter,
```

Comment:
Just making sure these don't have different values in the other variants, right? If they do they'll need to be set separately in variants.ts